projects/plugins/jetpack/tests/e2e/lib/env/playwright-environment.js

```diff
@@ -0,0 +1,155 @@
+const PlaywrightEnvironment = require( 'jest-playwright-preset/lib/PlaywrightEnvironment' ).default;
+const fs = require( 'fs' );
+const logger = require( '../logger' ).default;
+
+class PlaywrightCustomEnvironment extends PlaywrightEnvironment {
+	async setup() {
+		await super.setup();
+	}
+
+	async teardown() {
+		await super.teardown();
+	}
+
+	async handleTestEvent( event ) {
+		await super.handleTestEvent( event );
+
+		let testName = 'unknown_test';
+		let hookName = 'unknown_hook';
+
+		if ( event.test ) {
+			testName = `${ event.test.parent.name } - ${ event.test.name }`;
+		}
+
+		if ( event.hook ) {
+			hookName = `${ event.hook.type } - ${ event.hook.parent.name }`;
+		}
+
+		switch ( event.name ) {
+			case 'setup':
+				break;
+			case 'add_hook':
+				break;
+			case 'add_test':
+				break;
+			case 'run_start':
+				break;
+			case 'test_skip':
+				break;
+			case 'test_todo':
+				break;
+			case 'start_describe_definition':
+				break;
+			case 'finish_describe_definition':
+				break;
+			case 'run_describe_start':
+				break;
+			case 'test_start':
+				logger.info( `START TEST: ${ testName }` );
+				// await this.storeVideoFileName( testName );
+				break;
+			case 'hook_start':
+				logger.info( `START: ${ hookName }` );
+				// await this.storeVideoFileName( hookName );
+				break;
+			case 'hook_success':
+				logger.info( `SUCCESS: ${ hookName }` );
+				break;
+			case 'hook_failure':
+				logger.info( `FAILED: ${ hookName }` );
```

Comment:
```suggestion
				logger.info( `HOOK FAILED: ${ hookName }` );
```

Reply:
I see you marked it as resolved, but the suggestion was not accepted, nor declined.

Reply:
I didn't mark it as resolved, I pushed the suggestion using GH "Commit suggestion" button that apparently also marks it as resolved, as it applies the change exactly as suggested.